### PR TITLE
TM-44

### PR DESCRIPTION
**Pull request by aivis-vigo**
- i18n 
- some css styling

- en & lv.json in TM-45 are removed (from documentation they wasn't supposed to be there in angular 18 but if files wasn't placed in assets/i18n dir it wasn't working)

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,31 +1,57 @@
...
-          <a routerLink="create-task" routerLinkActive="active" ariaCurrentWhenActive="page">Create Task</a>
-        </li>
+  <!-- left side -->
+  <div>
+    <ul>
+      @if (userService.currentUserSig()) {
         <li>
-          <a routerLink="user-list" routerLinkActive="active" ariaCurrentWhenActive="page">User List</a>
+          <a routerLink="task-list" routerLinkActive="active" ariaCurrentWhenActive="page"
+             [translate]="'navigationBar.task-list'"></a>
```
Comment (wanpii) [inline src/app/app.component.html:8]: usually translation words are separated by . so for example, navigationBar.task.list

File: src/app/features/create-task/create-task.component.html [hunk 2]
```diff
@@ -46,6 +46,6 @@
     </div>
   }
 
-  <app-form-submit-button [formIsValid]="currentTask.valid" text="Submit"></app-form-submit-button>
+  <app-form-submit-button [formIsValid]="currentTask.valid" [text]="'button.create'"></app-form-submit-button>
```
Comment (wanpii) [inline src/app/features/create-task/create-task.component.html:49]: does this translate?

File: src/app/services/language.service.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  defaultLanguage: string = 'lv';
+  languageKey = 'language';
+
+  constructor(private translate: TranslateService) {
+    this.initializeTranslation();
+  }
+
+  initializeTranslation(): void {
+    const lang = localStorage.getItem(this.languageKey);
+    const initialLanguage = lang || this.defaultLanguage;
```
Comment (wanpii) [inline src/app/services/language.service.ts:18]: use ?? as it will fallback to the `this.defaultLanguage` in case lang is null

**Review by wanpii — CHANGES_REQUESTED** [review]

File: src/app/features/create-task/create-task.component.html (hunk 2 above)
Comment (aivis-vigo) [reply src/app/features/create-task/create-task.component.html:49]: yes

**Pull request merged**